tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -134,6 +134,7 @@ class DuckDBPyConnection:
     def list_filesystems(self) -> list: ...
     def load_extension(self, extension: str) -> None: ...
     def pl(self, chunk_size: int = ..., connection: DuckDBPyConnection = ...) -> polars.DataFrame: ...
+    def torch(self, connection: DuckDBPyConnection = ...) -> dict: ...
```

Comment:
Not a fan of this method name, but not sure of a better alternative. I wasn't even aware that the package for *py*torch was called *torch*?

Reply:
hm, could be `fetch_torch` but we do support `arrow` `df` `pl`

Reply:
Yeah, just annoying that torch is a verb **and** a noun